src/pages/Swap/index.tsx

```diff
@@ -65,7 +65,8 @@ import { computeFiatValuePriceImpact } from '../../utils/computeFiatValuePriceIm
 import { maxAmountSpend } from '../../utils/maxAmountSpend'
 import { computeRealizedPriceImpact, warningSeverity } from '../../utils/prices'
 import { supportedChainId } from '../../utils/supportedChainId'
-const ArrowContainer = styled.div`
+
+export const ArrowContainer = styled.div`
```

Comment:
nit: can we move this to a separate file? Ideally, we could have the `ArrowDown` component exported from `theme/components` and the fact that it uses `ArrowContainer` would just be an implementation detail. It's quite confusing to import just the container to wrap your icon.

Update: Maybe `styleds.tsx` can work too.

Reply:
can i do in a followup :D since this is already existing convention in the main Swap flow.

Reply:
Yes, please! Just make sure to not forget (I'll check hahaha jk)